docs/UserGuide.md

```diff
@@ -13,6 +13,25 @@ TABS aims to help project team leaders by
 
 Individuals with a fast typing speed will benefit more because TABS optimized for use via a **Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI).
 
+## Feature Overview
+
+The features of TABS are as such:
+1. Person management features - covering how to manage **Persons** in TABS.
+2. Group management features - covering how to manage **Groups**, which consists of **Members** in TABS.
+3. Task management features - covering how to manage **Tasks** amongst **Members**.
+4. Miscellaneous features - other features for convenience.
+
+<div markdown="span" class="alert alert-primary">
+:information_source: <b>Note</b>:
+
+* The terms _member_ and _person_ are different. A _person_ is someone who is registered in TABS. A _member_ is a _person_ who is part of a specific _group_.
+  * i.e Alex is a person, but he is not a member of Group 1. He is however a member of Group 2.
+* In order to add **Members**, the persons have to be present in TABS under **Contacts**.
+* On deletion of **Members** in **Groups**, the corresponding **Contacts** of the person will still remain.
```

Comment:
looks ok just need to be careful when we refer to the GUI must use contacts because it thats what it says but for reference use persons